CHANGES.md

```diff
@@ -1,3 +1,20 @@
+# 2.2.0
+
+## General
+- Lots of bug fixes and improvements
+- Tests are not built by default anymore, use `-DBUILD_TESTING=True`
```

Comment:
a bit late, but this was always the case.
the new thing is that the tests can _always_ be built regardless of cmake configuration with `make check`